herc/webservice.py

```diff
@@ -77,7 +80,7 @@ class schema(base):
     def get(self):
         """GET /schema
         Returns the JSON schema used to validate job submission requests."""
-        with open("data/schemas/jobsubmit.json", 'r', encoding="utf-8") as jschema:
+        with open(self.submit_schema, 'r', encoding="utf-8") as jschema:
```

Comment:
So, if we're going with @helgridly's suggestion, then this could be something like this for clarity:

```python
with open(get_data_path('data/schema/jobsubmit.json'), 'r', encoding="utf-8") as jschema:
```